### DISCUSSION TITLE
Alternate vehicles / different robots

### DISCUSSION THREAD

**Discussion by TooCoo** [microsoft/AirSim#3556] [Support Q&A] [answered]
Is there support or documentation for adding custom robots models to the simulator like the YouBot, TurtleBot, or Husky?

I imagine changing the mesh is fairly straight forward but using a different motion model, like diff-drive or omni would be harder.

**Comment by ahmed-elsaharti** [comment, marked as answer]
Hi @TooCoo, You're absolutely right.
changing the mesh is pretty easy (here's a quick [tutorial](https://drive.google.com/file/d/1-iOfJ8zO3YA4Mzc0ZkKwTM3L4--4RFtt/view?usp=sharing)). However, changing the motion model would require more in-depth digging especially if you're planning on changing the 'car' model since (AFAIK) that's based on UE's car physics.

**Reply by TooCoo** [reply]
Thanks for this, very helpful. 

I will dig deeper to see how I can change the car motion model, but for now this covers my needs.

**Comment by mrunaljsarvaiya** [comment]
Are there any updated on the general feature request of being able to add a vehicle with custom physics? In my case, I'd like to add a drone that has a payload attached to it via a ball joint.
